src/utils/size-utils.ts

```diff
@@ -0,0 +1,37 @@
+import { Theme } from '../theme';
+
+type Size = 'md' | 'sm';
+export const DEFAULT_SIZE: Size = 'sm';
+
+type SizeConfig = (theme: Theme, label: string, shouldMoveLeft?: boolean) => Record<Size, string>;
+
+export const getTextFieldSize: SizeConfig = (theme, label, shouldMoveLeft) => {
```

Comment:
this needs explanation as i dont fully get it by reading it

Reply:
ok, I will pass a description comment